scintillometry/utils/phase_utils.py

```diff
@@ -0,0 +1,135 @@
+# Licensed under the GPLv3 - see LICENSE
+
+import warnings
+
+import numpy as np
+import astropy.units as u
+import astropy.time as time
+
+try:
+    from .pint_utils import PintToas, make_toa_list
+    from pint.models import get_model
+except ImportError:
+    pass
+from .predictor import Polyco
+
+
+__all__ = ['PintPhase', 'PolycoPhase']
+
+
+class PintPhase(object):
+    """A utility class designated for computing phase using PINT.
+       Parameter
+       ---------
+       par_file : str
+           TEMPO/TEMPO2 style parameter fileself.
+       obs : str
+           Observatory name or observatory codeself.
+       obs_freq : float or `~astropy.units.Quantity`.
+           Observing frequency default units is 'MHz'
+       ephem : str
+           The solar system ephemeris version.
+       bipm_version : str
```

Comment:
I think examples here are useful. Alternatively, remove this (and `ephem` too?) and rely on defaults that can be overridden with `kwargs` (and tell where to look up what those are).